proto/p2p/p2p.proto

```diff
@@ -61,297 +61,357 @@ message Message {
-  // uptime is the primary network uptime percentage.
+  // Deprecated: uptime is now sent in Ping
+  // Uptime percentage on the primary network [0, 100]
   uint32 uptime = 1;
-  // subnet_uptimes contains subnet uptime percentages.
+  // Deprecated: uptime is now sent in Ping
+  // Uptime percentage on subnets
   repeated SubnetUptime subnet_uptimes = 2;
 }
 
-// The first outbound message that the local node sends to its remote peer
-// when the connection is established. In order for the local node to be
-// tracked as a valid peer by the remote peer, the fields must be valid.
-// For instance, the network ID must be matched and timestamp should be in-sync.
-// Otherwise, the remote peer closes the connection.
-// ref. "avalanchego/network/peer#handleVersion"
-// ref. https://pkg.go.dev/github.com/ava-labs/avalanchego/network#Network "Dispatch"
+// Version is the first outbound message sent to a peer when a connection is
+// established to start the p2p handshake.
+//
+// Peers must respond to a Version message with a PeerList message to allow the
+// peer to connect to other peers in the network.
+//
+// Peers should drop connections to peers with incompatible versions.
 message Version {
+  // Network the peer is running on (e.g local, testnet, mainnet)
   uint32 network_id = 1;
+  // Unix timestamp when this Version message was created
   uint64 my_time = 2;
+  // IP address of the peer
   bytes ip_addr = 3;
+  // IP port of the peer
   uint32 ip_port = 4;
+  // Avalanche client version
   string my_version = 5;
+  // Timestamp of the IP
   uint64 my_version_time = 6;
+  // Signature of the peer IP port pair at a provided timestamp
   bytes sig = 7;
+  // Subnets the peer is tracking
   repeated bytes tracked_subnets = 8;
 }
 
-// ref. https://pkg.go.dev/github.com/ava-labs/avalanchego/utils/ips#ClaimedIPPort
+// ClaimedIpPort contains metadata needed to connect to a peer
 message ClaimedIpPort {
+  // X509 certificate of the peer
   bytes x509_certificate = 1;
+  // IP address of the peer
   bytes ip_addr = 2;
+  // IP port of the peer
   uint32 ip_port = 3;
+  // Timestamp of the IP address + port pair
   uint64 timestamp = 4;
+  // Signature of the IP port pair at a provided timestamp
   bytes signature = 5;
+  // P-Chain transaction that added this peer to the validator set
   bytes tx_id = 6;
 }
 
-// Message that contains a list of peer information (IP, certs, etc.)
-// in response to "version" message, and sent periodically to a set of
-// validators.
-// ref. "avalanchego/network/network#Dispatch.runtTimers"
+// PeerList contains network-level metadata for a set of validators.
+//
+// PeerList must be sent in response to an inbound Version message from a
+// remote peer a peer wants to connect to. Once a PeerList is received after
+// a version message, the p2p handshake is complete and the connection is
+// established.
+
+// Peers should periodically broadcast PeerList messages to allow peers to
```

Comment:
Normally when I read "broadcast" I actually think the node is sending a peerlist message to _every_ connection it has. Could we instead say:
```suggestion
// Peers should periodically send PeerList messages to allow peers to
```